tools/gulp/packaging/rollup-helpers.ts

```diff
@@ -67,5 +72,19 @@ export function createRollupBundle(config: BundleConfig): Promise<any> {
     sourceMap: true
   };
 
+  // When creating a UMD, we want to exclude tslib from the `external` and `globals` bundle options
+  // so that it is inlined into the bundle.
+  if (config.format === 'umd') {
```

Comment:
It's just too much magic IMO. The `rollup-helpers` is a helper file that should be at least a bit generic.

Especially when we expose the package utilities (to for example flex-layout)